### PR TITLE
TW-29: Audit Logging

### PR DESCRIPTION
**Pull request by djmagee**
## What does this PR do?

Adds documentation for the first release of Audit Logging.

### CODE REVIEW

File: docs/cloud/how-to-manage-audit-logging.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+By using [Amazon Kinesis Data Firehose](https://docs.aws.amazon.com/firehose/latest/dev/what-is-this-service.html), you can route Temporal Audit Logs in [Amazon Simple Storage Service](https://docs.aws.amazon.com/s3/) (S3).
+We plan to release additional integrations.
+
+## Supported events
+
+The first release of Audit Logging supports Admin Operation events.
+
+### Admin Operation events
+
+Temporal logs operations at the account, user, and Namespace levels to ensure that your organization can audit and report changes to admin operations, when those changes occurred, and who made them.
```
Comment (jumbogo) [inline docs/cloud/how-to-manage-audit-logging.md:28]: Just use "namespace", no need to capitalize the first letter.

File: docs/cloud/how-to-manage-audit-logging.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+```
+
+The preceding code also prints the logs in the terminal.
+The following is a sample result.
+
+```json
+{
+  "emit_time": "2022-11-14T07:56:55Z",
+  "level": "LOG_LEVEL_INFO",
+  "user_email": "zhengbo@temporal.io",
```
Comment (jumbogo) [inline docs/cloud/how-to-manage-audit-logging.md:171]: Again, it's better to hide the real email address here

**Review by flossypurse — APPROVED** [review]
Stamping so you are not blocked when you get SME approval.

File: docs/cloud/how-to-manage-audit-logging.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+
+## Supported events
+
+The first release of Audit Logging supports Admin Operation events.
+
+### Admin Operation events
+
+Temporal logs operations at the account, user, and Namespace levels to ensure that your organization can audit and report changes to admin operations, when those changes occurred, and who made them.
+
+- Account
```
Comment (jumbogo) [inline docs/cloud/how-to-manage-audit-logging.md:30]: So for this list, the left hand side of ":" is the user operation, and the right hand side of ":" is the corresponding API that called in Temporal which emitting the logs.  Do we want to add this explanation in the doc? @djmagee

File: docs/cloud/how-to-manage-audit-logging.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+1. In the **Audit Logging** card, select **Configure Audit Logs**.
+1. On the **Audit Logging** page, choose your **Access method** (either **Auto** or **Manual**).
+   - **Auto:** Configure the AWS CloudFormation stack in your AWS account from the Cloud UI.
+   - **Manual:** Use a generated AWS CloudFormation template to set up Kinesis manually.
+1. In **Kinesis ARN**, paste the Kinesis ARN from your AWS account.
+1. In **Role name**, provide a name for a new IAM Role.
+1. In **Select an AWS region**, select the appropriate region for your Kinesis stream.
+
+If you chose the Auto access method, select **Save and launch stack**.
+The Audit Log sink is configured and ready to use.
```
Comment (jumbogo) [inline docs/cloud/how-to-manage-audit-logging.md:89]: After click Save and Launch stack, the user needs to enter a stack name in the next page and click "Create Stack" at the bottom right corner.  This page is also a AWS CloudFormation page, but is just a one click away from getting the whole thing set up.

File: docs/cloud/how-to-manage-audit-logging.md (hunk 3 above)
Comment (djmagee) [reply docs/cloud/how-to-manage-audit-logging.md:30]: Excellent idea; I'll do so.

**Review by jumbogo — APPROVED** [review]

**Pull request merged**